multimodal/environment.yml

```diff
@@ -0,0 +1,19 @@
+name: multimodal
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - pip=20.1.1
+  - python=3.7.3
+  - pip:
+    # Run requirements for hi-ml-azure
+    - azureml-sdk==1.36.0
+    - azureml-tensorboard==1.36.0
+    - conda-merge==0.1.5
+    - pandas==1.3.4
+    - param==1.12
+    - pysocks==1.6.0
+    - ruamel.yaml==0.16.12
+    - tensorboard==2.6.0
```

Comment:
Could we remove these dependencies for now? We've been refactoring the code to make sure it has as few dependencies as possible. Right now, the only dependencies should be einops, torch and transformers (torch only, not tensorflow).
We don't have any dependency on azureml yet, so maybe we could wait to add this and other dependencies in the future?

Reply:
removed now because the GPU tests are gone.